src/BloomBrowserUI/lib/jquery.myimgscale.js

```diff
@@ -72,6 +72,9 @@
         });
 
         function scale($img, $parent) {
+
+            $img.removeAttr("style");//hatton added. Else, change of options isn't always going to be reflected, as the old styles stick around
```

Comment:
We've vaguely talked about having a way to keep track of what we change in library code...did we ever come up with something satisfactory? I so, this change should be noted...if not, is this a good time to think of something?

In at least one case, I just added a ReadMe to the folder listing what I changed.